rosdep/python.yaml

```diff
@@ -5398,6 +5398,12 @@ python3-collada-pip:
   ubuntu:
     pip:
       packages: [pycollada]
+python3-colorama:
```

Comment:
This unfortunately collides with https://github.com/ros/rosdistro/pull/25547 .  I'll still review this, but I'm guessing this will have to be rebased once that goes in.

Reply:
Thanks. I have merged #25547

Reply:
I'm actually having a hard time reviewing this with the changes stacked. Since #25547 will be squash-merged this PR will need to be rebased onto the master branch after the other PR merges. Or since this PR is otherwise complete where the other is waiting for updated info. We could reset this PR and then request an update of #25547.